pkg/pillar/hypervisor/kvm.go

```diff
@@ -459,6 +455,8 @@ func (ctx kvmContext) Setup(status types.DomainStatus, config types.DomainConfig
 		return logError("failed to add kvm hypervisor loader to domain %s: %v", status.DomainName, err)
 	}
 
+	qemuOverHead := int64(config.Memory) * 1024 / 3
```

Comment:
While this is slightly better than the original, what I was thinking is that we should allocate this overhead based on the relative share of the particular VM compare to the rest of the VMs. IOW, once EVE's own overhead is taken into account (that # is available in the EVE's cgroup) we have a total of `T` RAM to be dolled out to VMs. Suppose we have n VMs with RAM allocations `V1`..`Vn` -- then we shall calculate the overhead per VM as: 
```
     (T - ∑Vi) * Vt
Ot = -------------
          ∑Vi
```